source/guide_offen.rst

```diff
@@ -0,0 +1,326 @@
+.. highlight:: console
+
+.. author:: Frederik Ring <hioffen@posteo.de>
+
+.. tag:: web
+.. tag:: analytics
+.. tag:: privacy
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/offen.png
+      :align: center
+
+##########
+Offen
```

Comment:
Please make sure the length of the `####` likes matches the length of `Offen` to get rid of build warnings.